spec/site_mercado/entities/payment_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+RSpec.describe SiteMercado::Entities::Order do
+  describe '.offline?' do
+    subject { payment.offline? }
+
+    context 'when offline' do
+      let(:payment) { build(:payment, :offline) }
+
+      it 'returns true' do
+        is_expected.to be_truthy
```

Comment:
Sei que nesse cenário está garantido por bater em um `#zero?`, mas só uma sugestão...

O be_truthy retorna true para outros resultados alem do true (1 por exemplo)
nesse caso eu faria o 

```suggestion
        is_expected.to be true
```